lib/driver.js

```diff
@@ -238,6 +244,39 @@ class YouiEngineDriver extends BaseDriver {
     
     this.proxydriver = await this.setupNewAndroidDriver(caps);
   }
+  
+  async setupNewMacDriver (caps) {
+    let macArgs = {
+      javascriptEnabled: true
+    };
+    let macdriver = new MacDriver(macArgs);
+    let capsCopy = _.cloneDeep(caps);
+    // Disabling the proxy CommandTimeout in the proxied driver since we are now handling it in the YouiEngine Driver
+    capsCopy.newCommandTimeout = 0;
+
+    await macdriver.createSession(capsCopy);
+
+    return macdriver;
+  }
+
+  async startMacSession (caps) {
+    logger.info("Starting a Mac proxy session");
+    this.proxyAllowList = TO_PROXY_MAC;
+    
+    this.proxydriver = await this.setupNewMacDriver(caps);
+  }
+  
+  startYIMacSession (caps) {  
+    
```

Comment:
`logger.info("Starting a YIMac proxy session");` to follow the same convention

Reply:
Well, we are not starting a proxy session.